tests/test_activities.py

```diff
@@ -58,45 +24,110 @@ def test_activies(test_db, workflow_file, config_dir, fixtures_dir):
 
     reset_db(test_db)
     load_fixture(test_db, "data_object_set.json")
-    if metatranscriptome:
-        load_fixture(test_db, "omics_processing_set_mt.json", col="omics_processing_set")
-    else:
-        load_fixture(test_db, "omics_processing_set.json")
+    load_fixture(test_db, "omics_processing_set.json")
+    load_fixture(test_db, "read_qc_analysis_activity_set.json")
 
-    wfs = load_workflows(config_dir / workflow_file)
+    wfs = load_workflows(workflows_config_dir / workflow_file)
     for wf in wfs:
         if not wf.type:
             continue
         # TODO: these tests are very sensitive to the exact content of the fixture files - need to be more robust
-        updated_fixtures = get_updated_fixture(wf, fixtures_dir)
+        updated_fixtures = get_updated_fixture(wf)
         if updated_fixtures:
             test_db[wf.collection].delete_many({})
             test_db[wf.collection].insert_many(updated_fixtures)
 
     acts = load_activities(test_db, wfs)
     # sanity check
-    assert acts is not None
-    if metatranscriptome:
-        # seq, rqc, assembly, annotation, expression
-        exp_num_acts = 5
-    else:
-        # seq, rqc, assembly, annotation, MAGs, read-based taxonomic annotation
-        exp_num_acts = 6
-    assert len(acts) == exp_num_acts
+    assert acts
+
+    # Omics and RQC share data_object_type for metagenome and metatranscriptome
+    # they can be distinguished by analyte category so we expect 1 of each
+    # for metagenome and metatranscriptome
     omics = acts[0]
     assert omics.type == "nmdc:OmicsProcessing"
     assert len(omics.children) == 1
     assert omics.children[0].type.lower() == "nmdc:ReadQCAnalysisActivity".lower()
+    rqc = acts[1]
+    assert rqc.type == "nmdc:ReadQcAnalysisActivity"
 
-
-def test_workflows(config_dir):
+@mark.parametrize(
+    "workflow_file", ["workflows.yaml", "workflows-mt.yaml"]
+    )
+def test_load_workflows(workflows_config_dir, workflow_file):
     """
     Test Workflow object creation
     """
-    wfs = load_workflows(config_dir / "workflows.yaml")
-    assert wfs is not None
+    metatranscriptome = False
+    if workflow_file == "workflows-mt.yaml":
+        metatranscriptome = True
+
+    shared_wf_names = ["Sequencing Noninterleaved", "Sequencing Interleaved"]
+    if metatranscriptome:
+        exp_num_wfs = 9
+        exp_wf_names = ["Metatranscriptome Reads QC",
```

Comment:
all of these should not run. It should be "Sequencing Noninterleaved" OR "Sequencing Interleaved", "Metatranscriptome Reads QC" OR  "Metatranscriptome Reads QC Interleave",  "Expression Analysis Antisense" OR 
                                      "Expression Analysis Sense" OR
                                      "Expression Analysis Nonstranded"
                                      
                                      exp_num_wfs = 5

Reply:
also lets remove "Sequencing Noninterleaved" OR "Sequencing Interleaved", so this should be 4.

Reply:
Added a docstring to load_workflows